ServiceWebsite/ServiceWebsite/Controllers/ParticipantController.cs

```diff
@@ -21,7 +21,7 @@ public class ParticipantController : Controller
     {
         private readonly IHearingsService _hearingService;
         private readonly IParticipantService _participantService;
-        private static readonly string strRegex = @"\b[A-Z]+(?:_[A-Z]+)+\b";
+        private static readonly string strRegex = @"(\b[A-Z]+(?:_[A-Z]+)+\b)|(\b[A-Z]+\b)";
```

Comment:
Is it not enough to simply have `[A-Z_]+`?

Reply:
the above will allow Keys that start/end with _.

Reply:
As Lionel said  [A-Z_]+?  will consider _Test or Test_ as Valid. What we got in the code will only allow 
TEST, TEST_KEY

Reply:
Yeah, you're right.